### ISSUE TITLE
Pascal VOC data split

### ISSUE THREAD

**Issue by ChenyuLin-Craig** [sandipan211/ZSD-SC-Resolver#19]
Hi @sandipan211, thanks for sharing this work!  I was recently trying to reproduce your results on PASCAL VOC but couldn't achieve such good experimental results.  I would like to know how you divided the PASCAL 2007 and 2012 datasets to make sure I didn't make a mistake in this step. Thanks for your time and consideration.

**Comment by sandipan211** [comment]
Hi @Craig-Lin616 ,
We downloaded the Pascal 2007 and 2012 datasets from the source website. For obtaining seen-unseen splits, we follow **[this](https://github.com/salman-h-khan/PL-ZSD_Release)** repository that proposes the 16/4 split. FYI, this is how my voc2007 folder looks like:
<img width="442" alt="image" src="https://github.com/sandipan211/ZSD-SC-Resolver/assets/30040548/11f7378b-89f4-45a5-bdcb-c88d26ac2b94">

and this is how my voc2012 folder looks like:
<img width="428" alt="image" src="https://github.com/sandipan211/ZSD-SC-Resolver/assets/30040548/6c08d36c-7900-4aa9-90fe-d35e614a8455">


Kindly note the following two points as well:
1. setting up the correct config file - in the news section in README, I have made it clear now which config file to run. When you connect the "data" field in the pascal voc config file with the above mentioned dataset folders, you will understand the folder organization and how the splits are being extracted. 
2. try using the pretrained models as uploaded by me for PASCAL-VOC, and then check whether the results are reproducible or not. Others have found this useful.
